build/config/BUILDCONFIG.gn

```diff
@@ -329,13 +331,14 @@ is_posix = !is_win && !is_fuchsia
 
 import("//chrobalt/build/configs/chrobalt.gni")
 # TODO(b/371589344): Fix android build configs.
-if (chrobalt && target_os != "android") {
+if (chrobalt) {
   import("//chrobalt/build/configs/initialize_variables.gni")
   import("//chrobalt/build/configs/variables.gni")
 }
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//starboard/build/config/includes:default_includes",
```

Comment:
Maybe there's a better way to do this?

Reply:
I'd leave it as a followup, and get this landed